### DISCUSSION TITLE
save_model, load_model

### DISCUSSION THREAD

**Discussion by bigluck07** [pycaret/pycaret#2913] [Q&A]
hi i just try to use for multitarget prob, but it doesn't suported, so i tried to solve one of each.

than i made a model and saved but I couldn't use it
```
def models(trn_df, trg):
    dt = trn_df.copy()
    dt[trg] = train_y[trg]
    reg = setup(data = dt, target=trg, train_size=.8, fold=2)
    comp = compare_models(sort = 'RMSE', n_select=3)
    evaluate_model(comp)
    tp1 = create_model(comp[0], cross_validation = False)
    tp2 = create_model(comp[1], cross_validation = False)
    tp3 = create_model(comp[2], cross_validation = False)
    tuned_tp1 = tune_model(tp1, optimize = 'RMSE', n_iter = 1)
    tuned_tp2 = tune_model(tp2, optimize = 'RMSE', n_iter = 1)
    tuned_tp3 = tune_model(tp3, optimize = 'RMSE', n_iter = 1)
    blender_specific = blend_models(estimator_list = [tuned_tp1, tp2, tuned_tp3],optimize = 'RMSE')
    final_model = finalize_model(blender_specific)
    pred = predict_model(final_model, data=test_x)
    return final_model, pred

Y_01, Y_01_pred = models(train_x, y_ls[0])
Y_01.save(model, path+'/Y_01.pt')
Y_01 = load_model(m_path+'/Y_01.pt')

predict_model(Y_01 , data=test_x)
```

BUT it made error 
am i somthing wrong?

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-29-2d246091ca0f>](https://localhost:8080/#) in <module>
----> 1 predict_model(Y_06, data=test_x)

4 frames
[/usr/local/lib/python3.7/dist-packages/pycaret/internal/preprocess.py](https://localhost:8080/#) in transform(self, dataset, y)
    362         # drop any columns that were asked to drop
    363         data.drop(columns=self.features_todrop, errors="ignore", inplace=True)
--> 364         data = data[self.final_training_columns]
    365 
    366         # also make sure that all the column names are string

AttributeError: 'DataTypes_Auto_infer' object has no attribute 'final_training_columns'

**Comment by tvdboom** [comment]
Please, open a new issue with a full reproducible example. And make sure to use the latest version of pycaret: `pip install --pre pycaret`

**Reply by bigluck07** [reply]
i will try thx :)
